Resources/Locale/en-US/store/currency.ftl

```diff
@@ -0,0 +1,9 @@
+store-currency-inserted = {CAPITALIZE(THE($used))} is inserted into the {THE($target)}.
+
+store-currency-free = Free
+store-currency-name-debugdollar = {$amount ->
+    [one] Debug Dollar
+    *[other] Debug Dollars
+}
+
+store-currency-name-telecrystal = TC
```

Comment:
This doesn't make much sense to me: one is an acronym, another is a full name. Shouldn't the bottom be "Telecrystals"?

Reply:
I suppose these are a bit of a misnomer. They aren't the display name of the currency, but rather the text used for displaying the price in the ui itself (i.e.: listing has a button that says [10 TC]). In that sense, it makes sense to use the shorter TC abbreviation for display, just like how the old uplink system has it.